src/components/post/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum PostFormType {
```

Comment:
should we call this PostFormType or PostFormVariant ? Since all the others are called XyVariant